internal/gcsx/random_reader.go

```diff
@@ -350,5 +370,25 @@ func (rr *randomReader) startRead(
 	rr.start = start
 	rr.limit = end
 
+	captureMetrics(ctx, rr.seeks)
```

Comment:
why pass the seeks here? Can we make a readType variable in this function, populate it based on the conditions in this function and then pass it to the captureMetrics?